vpplink/binapi/vpp_clone_current.sh

```diff
@@ -1,11 +1,34 @@
 #!/bin/bash
 set -e
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CACHE_DIR=$SCRIPTDIR/.cherries-cache
+
+function green ()
+{
+  printf "\e[0;32m$1\e[0m\n"
+}
+
+function commit_exists ()
+{
+  git show $1 > /dev/null 2>&1 || echo "no"
+}
+
 function git_cherry_pick ()
 {
 	refs=$1
-	git fetch "https://gerrit.fd.io/r/vpp" ${refs}
-	git cherry-pick FETCH_HEAD
+	mkdir -p $CACHE_DIR
+	CACHE_F=$CACHE_DIR/$(echo $refs |sed s@refs/changes/@@g |sed s@/@_@g)
+	if [ -f $CACHE_F ] && [ x$(commit_exists $(cat $CACHE_F)) != xnotfound ]; then
```

Comment:
Looks like commit_exists returns `no` on error not `notfound`?

Reply:
Right, thanks 😄